frontend/controller/actions/identity-kv.js

```diff
@@ -152,26 +153,27 @@ export default (sbp('sbp/selectors/register', {
   },
   // Preferences
   'gi.actions/identity/kv/fetchPreferences': async () => {
-    const { ourIdentityContractId } = sbp('state/vuex/getters')
-    if (!ourIdentityContractId) {
+    const identityContractID = sbp('chelonia/rootState').loggedIn?.identityContractID
+    if (!identityContractID) {
```

Comment:
[Elsewhere](https://github.com/okTurtles/group-income/pull/2137#discussion_r1667739696) in the contracts you do:

```js
sbp('state/vuex/state').loggedIn.identityContractID
```

So... why here are you using `sbp('chelonia/rootState')`? Alternatively, why [there](https://github.com/okTurtles/group-income/pull/2137#discussion_r1667739696) are you using `sbp('state/vuex/state')`?

Reply:
Maybe I should be consistent and use `chelonia/rootState`. The reason for this change is as follows:

If contracts are going to be asynchronous, using `chelonia/rootState` will be async (and also, it probably shouldn't be allowed, because it makes sandboxing ineffective). `state/vuex/state` likely won't exist in contracts (when using a SW) and should not be used either.

What I meant to do is to make these instances easy to find so that, in the future, Chelonia can pass globals to contracts when they are loaded. Those globals would be synchronous and would include environment information (e.g., the currently logged in user). By always using sbp('xxx').loggedIn, I ensure those uses are simple to find and replace.

Reply:
> Maybe I should be consistent and use `chelonia/rootState`

Yes please, consistency is always appreciated 🙏